### PR TITLE
Add libvips

### PR DESCRIPTION
**Pull request by thibaultmeyer**
Resolves: #6113
Signed-off-by: Thibault Meyer <meyer.thibault@gmail.com>

### CODE REVIEW

File: packages/libvips/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://libvips.github.io/libvips/
+TERMUX_PKG_DESCRIPTION="A fast image processing library with low memory needs"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="Thibault Meyer <meyer.thibault@gmail.com>"
+TERMUX_PKG_VERSION=8.10.5
+TERMUX_PKG_SRCURL=https://github.com/libvips/libvips/releases/download/v8.10.5/vips-8.10.5.tar.gz
```
Comment (xeffyr) [inline packages/libvips/build.sh:6]: Don't hardcode version in URL, use `${TERMUX_PKG_VERSION}` instead.

File: packages/libvips/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+TERMUX_PKG_SHA256=a4eef2f5334ab6dbf133cd3c6d6394d5bdb3e76d5ea4d578b02e1bc3d9e1cfd8
+TERMUX_PKG_DEPENDS="fftw, giflib, imagemagick, libexif, libjpeg-turbo, libtiff, libwebp"
```
Comment (xeffyr) [inline packages/libvips/build.sh:8]: Need `libexpat` as build log error has shown.

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: packages/libvips/build.sh (hunk 1 above)
Comment (thibaultmeyer) [reply packages/libvips/build.sh]: Done

File: packages/libvips/build.sh (hunk 2 above)
Comment (thibaultmeyer) [reply packages/libvips/build.sh]: Done

**Comment by Grimler91** [comment]
Nice work!

**Pull request merged**